packages/server/lib/modes/run.ts

```diff
@@ -492,6 +492,12 @@ async function waitForBrowserToConnect (options: { project: Project, socketId: s
       openProject.updateTelemetryContext(JSON.stringify(telemetry.getActiveContextObject()))
     }
 
+    // since we aren't going to be opening a new tab,
+    // we need to tell the protocol manager to reconnect to the existing browser
+    if (options.browser.family === 'chromium') {
```

Comment:
Let's do this if the protocolManager is there rather than relying on the browser family.

Reply:
Updated: [d586c22](https://github.com/cypress-io/cypress/pull/27659/commits/d586c225ca0c4e4d2a4344d43d2cddd0b1ffb091)